### ISSUE TITLE
Support pre-defined sets of keys

### ISSUE THREAD

**Issue by akhon** [Soluto/kamus#150]
We have to use predefined sets of keys to encrypt/decrypt secrets to get keys control hardened.
So, would be awesome to use full arn (we use AWS KMS) sets of keys in values and not create new ones

**Comment by omerlh** [comment]
Interesting. We can support this, but this will make the encryption weaker - currently, there is one key per service account/namespace combination. How would you imagine this setup? Having a static mapping of service accounts/namespace to ARNs?

**Comment by akhon** [comment]
> Having a static mapping of service accounts/namespace to ARNs?

yep, that would be a good starting point, or might be like this: use default KMS key + static mapping of namespaces-keys

**Comment by omerlh** [comment]
Interesting feature. A PR will be appreciated here :)
